rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProviderImpl.java

```diff
@@ -324,6 +328,12 @@ public void setLastRetiredFederationP2SHScript(Script lastRetiredFederationP2SHS
         this.lastRetiredFederationP2SHScript = lastRetiredFederationP2SHScript;
     }
 
+    @Override
+    public void setSvpFundTransactionUnsignedHash(Sha256Hash hash) {
+        this.svpFundTransactionUnsignedHash = hash;
+        isSvpFundTransactionUnsignedHashSet = true;
```

Comment:
nit: I would also reference using `this` here

Reply:
Thanks. I will update it in the next pr.